tests/sentry/api/endpoints/test_project_rules_configuration.py

```diff
@@ -182,7 +177,7 @@ def test_sentry_app_alert_rules(self, mock_sentry_app_components_preparer):
 
     def test_issue_type_and_category_filter_feature(self):
         response = self.get_success_response(self.organization.slug, self.project.slug)
-        assert len(response.data["actions"]) == 10
+        assert len(response.data["actions"]) == 12
```

Comment:
Can you add tests to https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/test_ticket_rules.py too for these?

Reply:
ok, I'm gonna first split the tests by integration here: https://github.com/getsentry/sentry/pull/59065